src/DirectChildElementMatcher.php

```diff
@@ -54,6 +58,7 @@ protected function matchesSafelyWithDiagnosticDescription( $item, Description $m
 			return $directChildren->length !== 0;
 		}
 
+		$child = null;
```

Comment:
This line is technically dead code.

Reply:
Yeah, it felt cleaner than adding some sort of phan/other type suppression.